qa/tasks/cephfs/test_subvolume.py

```diff
@@ -29,6 +30,9 @@ def setup_test(self):
     def cleanup_test(self):
         self.mount_a.run_shell(['rm', '-rf', 'group'])
 
+    def _fs_cmd(self, *args):
```

Comment:
Is this helper _really_ required?

Reply:
the helper does seem helpful to me

Reply:
Well, its just doing

> return self.get_ceph_cmd_stdout("fs", *args)

so basically just calling self.get_ceph_cmd_stdout("fs", ...) would do the job, isn't it?

Reply:
I see `def _fs_cmd` in `test_volumes.py` and `test_snap_schedules.py` too. I think this redundancy can be avoided if we just define this in `qa/tasks/cephfs/cephfs_test_case.py` in class `CephFSTestCase` since all these test cases file do import `CephFSTestCase`

Reply:
removed `def _fs_cmd()` helper